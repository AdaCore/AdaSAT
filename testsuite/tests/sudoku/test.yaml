project_file: prj.gpr
executable: sudoku
args_matrix:
    - ["sudoku_grids/empty.txt"]
    - ["sudoku_grids/sudoku_15.txt"]
    - ["sudoku_grids/sudoku_ai_escargot.txt"]
    - ["sudoku_grids/sudoku_ai_2.txt"]
