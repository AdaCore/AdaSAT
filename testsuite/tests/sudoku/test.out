[sudoku_grids/empty.txt]
Solved
 1 2 3 4 5 6 7 8 9
 4 5 6 7 8 9 1 2 3
 7 8 9 1 2 3 4 5 6
 2 1 4 3 6 5 8 9 7
 3 6 5 8 9 7 2 1 4
 8 9 7 2 1 4 3 6 5
 5 3 1 6 4 2 9 7 8
 6 4 2 9 7 8 5 3 1
 9 7 8 5 3 1 6 4 2

[sudoku_grids/sudoku_15.txt]
Solved
 7 9 2 5 6 8 1 4 3
 4 5 3 2 1 9 8 6 7
 8 6 1 3 7 4 9 5 2
 6 2 5 8 9 3 7 1 4
 3 7 9 1 4 2 6 8 5
 1 4 8 7 5 6 2 3 9
 2 8 4 9 3 1 5 7 6
 9 3 7 6 8 5 4 2 1
 5 1 6 4 2 7 3 9 8

[sudoku_grids/sudoku_ai_escargot.txt]
Solved
 1 6 2 8 5 7 4 9 3
 5 3 4 1 2 9 6 7 8
 7 8 9 6 4 3 5 2 1
 4 7 5 3 1 2 9 8 6
 9 1 3 5 8 6 7 4 2
 6 2 8 7 9 4 1 3 5
 3 5 6 4 7 8 2 1 9
 2 4 1 9 3 5 8 6 7
 8 9 7 2 6 1 3 5 4

[sudoku_grids/sudoku_ai_2.txt]
Solved
 8 1 2 7 5 3 6 4 9
 9 4 3 6 8 2 1 7 5
 6 7 5 4 9 1 2 8 3
 1 5 4 2 3 7 8 9 6
 3 6 9 8 4 5 7 2 1
 2 8 7 1 6 9 5 3 4
 5 2 1 9 7 4 3 6 8
 4 3 8 5 2 6 9 1 7
 7 9 6 3 1 8 4 5 2

